templates/lib/template.json

```diff
@@ -25,11 +25,5 @@
             "default": "/${LIBRARY_NAME}/-/_/",
             "auto": true
         }
-    },
```

Comment:
See other comment. This change needs to be reverted.

Reply:
Fixed